README.md

```diff
@@ -1,89 +1,494 @@
+as soon as possible!
+But _definitely_ "stay in school"
+and learn everything you can,
+because life it's much easier
+if you tick the box of "school" first.
+If you are still in high school
+and feel that your creativity
+is _not_ being rewarded,
+don't worry or lose hope,
+it will be soon enough! </small>
+
 
 ## How?
 
 ### Prerequisites
 
-+ Read (_and "star"_) `this` repo to show you have (_have read_)
++ **Read** (_and "star"_) `this` repo to show you have (_have read_)
 and are interested in the apprenticeship.
-+  Read (_and "star"_) the dwyl contributing guide:
++ **Read** (_and "star"_) https://github.com/dwyl/start-here
+to show you _agree_ with @dwyl's
+[mission](https://github.com/dwyl/start-here/blob/master/mission.md),
+[manifesto](https://github.com/dwyl/start-here/blob/master/manifesto.md),
+manifesto
++  ***Read*** (_and "star"_) the dwyl **contributing guide**:
 https://github.com/dwyl/contributing
-to confirm that you have read and understood our "team workflow".
-+ Previous contribution to a dwyl project
+to confirm that you have read and understood our "**team workflow**".
++ ***Answer*** the questions see:
+[start-here/**questions.md**](https://github.com/dwyl/start-here/blob/master/questions.md)
++ ***Contribute*** to a dwyl project
 or tutorial(s): https://github.com/dwyl?utf8=%E2%9C%93&q=learn
 (_a Pull Request with an improvement to the learning materials_)
-+ Answer the "AMA" questions see: `/ama`
+See: What is a "_Previous Contribution_" question/answer below.
 
-<br /><br /><br />
+### _Apply_!
 
-## Way More Questions & Answers!
+Send us an email: **hello@dwyl.com** with the subject: "Apprenticeship".
+Describe _why_ you want to join our Apprenticeship program and
+include a link to your GitHub profile.
+We will take it from there.
 
-### Q: What is a "Previous Contribution" to a @dwyl Project?
+<br /><br /><br />
+
+## Frequently Asked Questions & _Answers_!
 
-Contribution to a dwyl project can range from:
-+ asking a question (_seeking clarification_) in a GitHub Issue
-+ reporting a bug/issue on one of our projects or tutorials
-+ Creating a Pull Request that _fixes_ the bug/issue you reported
-and or adds the question/answer to a relevant readme.
 
 ### Q: How long is a day?
 [asked by @ZooeyMiller in #2](https://github.com/dwyl/apprenticeship/issues/2)
 
-Aim to start at 09:00. If you are prefer to start earlier,
-just let someone know you are available/online!
+You should aim to spend **6 hours** per day on _deep work_.<sup>1</sup>
+You are responsible for managing/tracking your time.
+Divide your time between learning, researching, writing, coding, editing
+and other creative work.
+Aim to **start** your day no later than **08:50 GMT**
+so that you are always "on time"
+for the daily morning meeting.
+
+If you are prefer to start earlier, do it!
+Some of us start our day at 06:00 so we get 3h
+of focus time *before* 09:00!
+Just let someone know you are available/online
```

Comment:
Do they need to let someone know? My reasoning for getting up early (when I manage it!) is precisely because no one knows or expects me to be online so it is uninterrupted time (both remote and from people in my location). Would it be enough for them to track their time (which would eventually be visible in our app) and follow the communication protocols around issue updates?

Reply:
@iteles one of the _highest_ features on my List of Desired Features for the App: https://github.com/dwyl/app/issues/262#issuecomment-577868398 is "Focus Mode". 
An apprentice "informing" people that they are woking will be as simple as them starting a timer on their task for the morning and activating "Focus Mode" so that their peers know they are working and not to be interrupted. 
I haven't _specified_ that here because we don't _have_ that feature yet.
Once we have that feature in our App, we can update this. 👍